api-gateway/src/main/resources/static/scripts/inventory-list/inventory-list.controller.js

```diff
@@ -13,6 +13,13 @@ angular.module('inventoryList')
 
         getInventoryList()
 
+
+                $http.get('api/gateway/inventory').then(function (resp) {
+                    self.inventoryList = resp.data;
+                    console.log("Resp data: " + resp.data)
+                    console.log("inventory list: " + self.inventoryList)
+                });
+
```

Comment:
I think this might break the pagination because you need to add the query params for the page and size of the list. Can you tell me if the pagination still work

Reply:
![image](https://github.com/cgerard321/champlain_petclinic/assets/91092017/318ff659-c3f2-4bd9-858e-8defa6697ee5)
![image](https://github.com/cgerard321/champlain_petclinic/assets/91092017/89e7b167-0cb8-4c69-9fd9-ad6b3e136a93)
After running your code. The pagination that you made still works fine.

Reply:
If it works great, but I don't think you need to keep it. The getInventoryList() is called without any field so it is doing the get all inventories.

Reply:
I will take it into consideration and talk with the team members to decide whether or not we should remove it.
Thank you.